src/app/portlets/dot-apps/dot-apps-export-dialog/dot-apps-export-dialog.component.html

```diff
@@ -0,0 +1,22 @@
+<dot-dialog
+    class="p-fluid"
+    *ngIf="showExportDialog"
+    [header]="'apps.confirmation.export.header' | dm"
+    [visible]="showExportDialog"
+    [actions]="dialogExportActions"
+    [appendToBody]="true"
+    (hide)="closeExportDialog()"
+>
+    <form [formGroup]="form" novalidate>
+        <input
+            dotAutofocus
+            autocomplete="new-password"
+            formControlName="password"
+            name="export-password"
+            pInputText
+            type="password"
+            placeholder="{{ 'apps.confirmation.export.password.label' | dm }}"
```

Comment:
```suggestion
            [placeholder]="'apps.confirmation.export.password.label' | dm"
```

https://angular.io/guide/interpolation#expression-context